docs/storage-surface.rst

```diff
@@ -0,0 +1,916 @@
+``gcloud.storage`` API
+======================
+
+Connection / Authorization
+--------------------------
+
+- Inferred defaults used to create connection if none configured explicitly
+
+  - credentials (derived from GAE / GCE environ if present).
+
+  - ``project`` (derived from GAE / GCE environ if present).
+
+  - ``bucket`` (derived from GAE / GCE environ if present).
+
+- By calling methods which require authentication
+
+  .. code-block:: python
+
+   >>> from gcloud import storage
+   >>> bucket = storage.get_bucket('my-bucket')
+
+  the default connection "just works" out of the box and will be lazily
+  loaded and stored after the first use.
+
+- Set defaults in a declarative fashion
+
+  .. code-block:: python
+
+   >>> storage.set_defaults()
```

Comment:
Look down 3 lines. It's much less useful without arguments now that we have lazy-loading. We've even discussing removing it completely.